File: Andriod/Demo/Demo/app/src/main/java/com/example/demo/SecondActivity.java
package com.example.demo;

import android.os.Bundle;
import android.os.Environment;
import android.util.Log;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import java.io.File;

import butterknife.BindView;
import butterknife.ButterKnife;

public class SecondActivity extends AppCompatActivity implements View.OnClickListener{

    @BindView(R.id.toolbar)
    View toolbar;
    ImageView back;
    TextView tvHeading;

    @Override
    protected void onCreate(Bundle saveInstanceState){
        super.onCreate(saveInstanceState);
        setContentView(R.layout.activity_second);
        ButterKnife.bind(this);
        settoolbar();

        String path = Environment.getExternalStorageDirectory()+File.separator+"DCIM";
        Log.v("Files", "Path: " + path);
        File directory = new File(path);
        File files[] = directory.listFiles();
        Log.v("Files", "Size: "+ files.length);
        for (int i = 0; i < files.length; i++)
        {
            Log.v("Files", "FileName:" + files[i].getName());
        }

    }

    public void settoolbar(){
        back=toolbar.findViewById(R.id.ivBack);
        back.setImageResource(R.drawable.ic_arrow_back_black_24dp);
        back.setOnClickListener(SecondActivity.this);
        tvHeading=toolbar.findViewById(R.id.tvHeading);
        tvHeading.setText("Second Activity");
    }

    @Override
    public void onClick(View view) {
        switch (view.getId()){
            case R.id.ivBack:
                finish();
                break;
            default:
                break;
        }
    }
}

File: Kotlin/CerylMap/app/src/main/java/com/cerylmap/activity/SecondActivity.kt
package com.cerylmap.activity

import android.app.Dialog
import android.os.Bundle
import android.view.Window
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.cerylmap.R
import kotlinx.android.synthetic.main.activit_second.*

class SecondActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activit_second)


        btnDialog.setOnClickListener {
            var dialog:Dialog?=null
            dialog !!.requestWindowFeature(Window.FEATURE_NO_TITLE)
            dialog .setCancelable(false)
            dialog .setContentView(R.layout.custom_layout)
            val body = dialog .findViewById(R.id.tvTitle) as TextView
            body.text = "Title of the Dialog"
            val yesButton = dialog .findViewById(R.id.btnDialogClick) as Button
            yesButton.setOnClickListener {
                dialog .dismiss()
            }

        }

    }


}